typescript/packages/well-log-viewer/src/SyncLogViewer.tsx

```diff
@@ -540,6 +540,7 @@ class SyncLogViewer extends Component<SyncLogViewerProps, State> {
 
         if (this._isMounted) this.setState({ primaryAxis: value });
     }
+    /*
```

Comment:
If the code is commented out, can it be removed as to not pollute the code?

Reply:
I have removed it